ci-operator/config/openshift-kni/eco-ci-cd/.config.prowgen

```diff
@@ -10,4 +10,4 @@ slack_reporter:
                         logs> {{end}}'
   job_names:
   - eco-ci-cd
-  
+  - openshift-kni-eco-ci-cd-main-cnf-core-4.16
```

Comment:
Based on the [documentation](https://docs.ci.openshift.org/docs/how-tos/notification/), you can either remove this job since it uses the same 'as' field as 'eco-ci-cd', or rename the 'as' field to something different.